index.html

```diff
@@ -0,0 +1,208 @@
+  <link href="https://fonts.googleapis.com/css2?family=Quicksand:wght@700&display=swap" rel="stylesheet">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=GFS+Didot&family=Lalezar&family=Source+Sans+Pro:wght@700&family=Varta&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=GFS+Didot&family=Source+Sans+Pro:wght@700&family=Varta&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="home.css">
+  <link rel="stylesheet" href="root.css">
+</head>
+<body>
+  <header>
+    <div class="container-fluid header-bg position-fixed">
+      <div class="d-flex justify-content-end">
+        <div class="d-flex justify-content-between align-items-center gap-3 me-5">
+          <a href="#/"><i class="fa fa-facebook white pb-2"></i></a>
+          <a href="#/"><i class="fa fa-twitter white pb-2"></i></a>
+          <span class="white header-text">English</span>
+          <span class="white header-text">My Page</span>
+          <span class="white header-text">Logout</span>
+        </div>
+      </div>
+    </div>
+  </header>
+  <nav>
+    <div class="container-fluid position-fixed nav-container">
+      <div class="hamburger">
+        <div class="bar"></div>
+        <div class="bar"></div>
+        <div class="bar"></div>
+      </div>
+      <div class="row align-items-center">
+          <div class="col-12 col-md-5 d-flex justify-content-center align-items-center pt-md-0 pb-md-0">
+            <a href="index.html" class="d-flex justify-content-center" ><img src="img/logo.png" class="nav-icon pt-md-0"></a>
+          </div>
+          <div class="col col-md-4 d-flex align-items-center fw-bold">
+            <ul class="nav-links mb-0">
+              <a href="about.html"><li>About</li></a>
+              <a href=""><li>Program</li></a>
+              <a href=""><li>Join</li></a>
+              <a href=""><li>Sponsor</li></a>
+              <a href=""><li>News</li></a>
+            </ul>
+          </div>
+          <div class="col-md-3 align-items-center justify-content-start d-none d-md-flex ps-5">
+            <p class="p-1 m-1 campaign-text">CC Campaign</p>
+          </div>
+      </div>
+    </div>
+  </nav>
+  <section>
+    <div class="container-fluid ps-md-5" id="intro">
+      <div class="row ps-md-5 ms-md-5 justify-content-start">
+        <div class="col-12 col-md-7 ms-md-5 mt-5 pt-5">
+          <h1 class="fs-5 mb-1 intro-hello">"WE ARE NOW IN MYANMAR!"</h1>
```

Comment:
- [ ] nice job. please like the previous reviewer noted, each page should have one h1 that stands as the principal header and then each section should make use of the next level hierarchy tag h2 for the main heading of the section or article. please kindly change the 'we are now in myanmar' to an h2 tag